test/powershell/Language/Scripting/Dynamicparameters.Tests.ps1

```diff
@@ -0,0 +1,68 @@
+﻿Describe "Dynamic parameter support in script cmdlets." -Tags "CI" {
+    BeforeAll {
```

Comment:
We should also add some tests where the script cmdlet returns a bindable object, basically a class that has properties with the Parameter attribute.